source/helpers/dom-utils.ts

```diff
@@ -73,8 +73,29 @@ export const unhighlightTab = (tabElement: Element): void => {
 	tabElement.removeAttribute('aria-current');
 };
 
-export const assertNodeContent = ({textContent}: Node, includes: string): void => {
-	if (!textContent!.includes(includes)) {
-		throw new TypeError(`Expected node including "${includes}", found ${textContent!.trim()}`);
+const matchString = (matcher: RegExp | string, string: string): boolean =>
+	typeof matcher === 'string' ? matcher === string : matcher.test(string);
+
+const escapeMatcher = (matcher: RegExp | string): string =>
+	typeof matcher === 'string' ? `"${matcher}"` : String(matcher);
+
+// eslint-disable-next-line @typescript-eslint/ban-types -- Nodes may be exactly `null`
+export const assertNodeContent = <N extends Text | ChildNode>(node: N | null, expectation: RegExp | string): N => {
+	if (!node || !(node instanceof Text)) {
+		console.warn('TypeError', node);
```

Comment:
I wish there was a way to attach data to errors and have the browser actually display it. `error.PROP` is basically inaccessible from the console, let alone _visible_